paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -75,18 +73,11 @@ internal fun CustomerSheetScreen(
                         PaymentSheetContentPadding()
                     }
                     is CustomerSheetViewState.AddPaymentMethod -> {
-                        if (customerSheetACHv2.isEnabled) {
-                            AddPaymentMethodWithPaymentElement(
-                                viewState = viewState,
-                                viewActionHandler = viewActionHandler,
-                                displayForm = displayAddForm,
-                            )
-                        } else {
-                            AddPaymentMethod(
-                                viewState = viewState,
-                                viewActionHandler = viewActionHandler,
-                            )
-                        }
+                        AddPaymentMethodWithPaymentElement(
```

Comment:
Should we just rename this to `AddPaymentMethod`?

Reply:
Probably.